gobase/command/command.go

```diff
@@ -34,11 +35,11 @@ func Command(c *cli.Context) error {
 		defer postgres.Close()
 		switch secondArgs {
 		case "fresh":
-			db.Fresh(postgres)
+			db.Fresh(postgres, thirdArgs)
```

Comment:
thirdArgs has no meaning at all

Reply:
Resolved in PR https://github.com/wincentrtz/gobase/pull/5